Adding Project Subdirectories (#3090)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/projects.md

For complete details on project configurations, see [dbt_project.yml](/reference/dbt_project.yml).

## Project subdirectories

> The Project Subdirectory option in dbt Cloud allows you to specify a subdirectory within your git repository that dbt should use as the root directory for your project. This can be useful in situations where you have multiple dbt projects within a single repository, or when you want to organise your dbt project files into subdirectories for easier management.

Reviewer: ```suggestion
You can use the Project subdirectory option in dbt Cloud to specify a subdirectory in your git repository that dbt should use as the root directory for your project. This is helpful when you have multiple dbt projects in one repository or when you want to organize your dbt project files into subdirectories for easier management.
```

> To use the Project Subdirectory option in dbt Cloud, follow these steps:

Reviewer: ```suggestion
To use the Project subdirectory option in dbt Cloud, follow these steps:
```

> 1. **Click** the Settings dropdown on the top navigation menu and **Click** Account Settings

Reviewer: ```suggestion
1. Click on the cog icon on the upper right side of the page and click on **Account Settings**.
```

> <img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">

Reviewer: i can't see the image here. to add images:
- use `<Lightbox src="/img/docs/your_image-name.jpg" title="Add your title here"/> `
- in that same branch, add your images probably in this [location](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/static/img/docs/building-a-dbt-project) of the docs repo (please use jpg)

> <img width="379" alt="project-subdirectory-01" src="https://user-images.githubusercontent.com/114556261/228104923-591cd791-6f38-4a10-a983-7d85cd18f0b2.png">

Reviewer: sugg removing img for this step

```suggestion
```

> 2. **Click** on a Project to configure Project Subdirectory

Reviewer: ```suggestion
2. Under **Projects**, select the project you want to configure as a project subdirectory.
```

> 3. **Click** on Edit on the sliding pane

Reviewer: ```suggestion
3. Select **Edit** on the lower right-hand corner of the page.
```

> <img width="1409" alt="project-subdirectory-02" src="https://user-images.githubusercontent.com/114556261/228104897-954e280b-cb68-4f7c-be3a-ceba5d9b90e3.png">

Reviewer: sugg removing this screenshot

```suggestion
```

> 4. **Enter** the name of the subdirectory in Project Subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory. 

Reviewer: ```suggestion
4. In the **Project subdirectory** field, add the name of the subdirectory. For example, if your dbt project files are located in a subdirectory called `<repository>/finance`, you would enter `finance` as the subdirectory. 
```

Reviewer: love this example ✨

> You can also reference nested subdirectories. For example, if your dbt project files are located in `<repository>/teams/finance`, you would enter `teams/finance` as the subdirectory. Please note: you do not need leading or trailing `/` in the Project Subdirectory parameter. **Click** the Save button when complete.

Reviewer: ```suggestion
* You can also reference nested subdirectories. For example, if your dbt project files are located in `<repository>/teams/finance`, you would enter `teams/finance` as the subdirectory. Please note: you do not need leading or trailing `/` in the Project subdirectory parameter. 

5. Click **Save** when you've finished.
```

> <img width="660" alt="project-subdirectory-03" src="https://user-images.githubusercontent.com/114556261/228117931-f2651529-ebf3-4f19-bd33-14e299810a6f.png">

Reviewer: ```suggestion
<Lightbox src="/img/docs/building-a-dbt-project/project-subdirectory.jpg" width="75%"  title="Entering `finance` in the Project subdirectory field."/>
```

> Once you have configured the Project Subdirectory option, dbt Cloud will use the specified subdirectory as the root directory for your dbt project. This means that any dbt commands that you run in dbt Cloud, such as dbt run or dbt test, will operate on the files within the subdirectory that you specified. If there is no `dbt_project.yml` file detected in the Project Subdirectory, you will be prompted to initialise the dbt project.

Reviewer: ```suggestion
After configuring the Project subdirectory option, dbt Cloud will use it as the root directory for your dbt project. This means that dbt commands, such as `dbt run` or `dbt test`, will operate on files within the specified subdirectory. If there is no dbt_project.yml file in the Project subdirectory, you will be prompted to initialize the dbt project.
```
